a10_neutron_lbaas/v2/handler_listener.py

```diff
@@ -74,14 +74,20 @@ def _set(self, set_method, c, context, listener):
             except Exception as ex:
                 LOG.exception(ex)
 
-            if bindings and len(bindings) > 0:
+            unbound = any([not bool(x.status) for x in bindings])
```

Comment:
No need for the `[` ... `]`s. `any` works on iterables, no need to construct a list from the iterable.